Plugin page update (#563)
PennyLaneAI/pennylane-qiskit: doc/index.rst

  :end-before: header-end-inclusion-marker-do-not-remove

> Once the PennyLane-Qiskit plugin is installed, the Qiskit devices

Reviewer: The "Backends" section on this page also feels outdated - it includes passing a string backend, which is only relevant for the Aer device now. It also claims that PennyLane chooses the `aer_simulator` as the default backend if none is specified, which is now only true for the Aer device.

Backends
~~~~~~~~

Qiskit devices have different **backends**, which define which actual simulator or hardware is 

> used by the device. We recommend passing in a backend instance rather than a string with the 
> backend name to the devices. 

Reviewer: ```suggestion
used by the device. A backend instance should be and passed to the device when initializing.
```

Since the string method is only really relevant for the `aer` device now, and will potentially be deprecated soon, I think we should just avoid mentioning it entirely in the general index.

Different simulator backends are optimized for different purposes. To change what backend is used, 
a simulator backend can be defined as follows:

.. code-block:: python

    from qiskit_aer import UnitarySimulator

    dev = qml.device('qiskit.aer', wires=<num_qubits>, backend=UnitarySimulator())

.. note::

    For ``'qiskit.aer'``, PennyLane chooses the ``aer_simulator`` as the default backend if no 
    backend is specified. For more details on the ``aer_simulator``, including available backend 
    options, see `Qiskit Aer Simulator documentation <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.AerSimulator.html#qiskit_aer.AerSimulator.run>`_.

    Occassionally, you may see others pass in a string as a backend. For example:

    .. code-block:: python

        dev = qml.device('qiskit.aer', wires=<num_qubits>, backend='unitary_simulator')

    At the time of writing, this is still functional. However, this will soon be deprecated and may 

>     not function as intended. To ensure accurate results, we recommend passing in a backend instance.

Reviewer: This section of the note is specific enough to the `aer` device (and to a UI we are moving away from) that I would suggest moving it to the `aer` device page
